### DISCUSSION TITLE
Netflow stats?

### DISCUSSION THREAD

**Discussion by G0nz0uk** [pfelk/pfelk#335] [General]
Hello,

I've installed ElastiFLow using your scrips and it's pulling in information from my OPNsense firewall, it great.

How can I get Netflow stats coming in as well?

Thanks

**Comment by a3ilson** [comment]
@G0nz0uk - can you clarify? The scripts for pfELK do not install ElastiFlow.

**Reply by G0nz0uk** [reply]
I'm new to ELK, so I've been reading up on it.  I see now your wonderful scripts build the ELK stack and ElastiFlow is extra and needs to be install on top.
I'd love to get my OPNsense flows showing up, do you have any instructions on how to add this great feature?

**Reply by a3ilson** [reply]
@G0nz0uk - Unfortunately, we do not at the moment.
